reactive/spark.py

```diff
@@ -123,5 +119,3 @@ def client_present(client):
 @when_not('spark.started')
 def client_should_stop(client):
     client.clear_spark_started()
-
-
```

Comment:
We do not have much unit testing. We have integration tests that are mostly part of the bundles.